src/k8s/pkg/k8sd/app/cluster_util.go

```diff
@@ -33,6 +33,27 @@ func setupKubeconfigs(s state.State, kubeConfigDir string, securePort int, pki p
 
 }
 
+func SetupControlPlaneKubeconfigs(s state.State, kubeConfigDir string, securePort int, pki pki.ControlPlanePKI) error {
```

Comment:
this is an exact duplicate of the function `setupKubeconfigs` above. 
Also this is not used anywhere.